src/main/java/com/epam/asset/tracking/service/UserService.java

```diff
@@ -1,5 +1,9 @@
 package com.epam.asset.tracking.service;
 
-public interface UserService {
+import com.epam.asset.tracking.domain.BusinessProvider;
+import com.epam.asset.tracking.exception.InvalidUserException;
 
+public interface UserService {
```

Comment:
Methods in this interface use and return BusinessProvider objects. Shouldn't they be better in a BusinessProvider service?

Reply:
Yes they should, and I have already make the change